Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -349,6 +349,11 @@ public void onError(String id, VungleException error) {
                         "for Placement ID: " + id);
             }
         }
+
+        @Override
+        public void onAdViewed(String id) {
+            //no-op for now(6.8.1)
```

Comment:
Is this means the `onAdViewed` callback has nothing to do now?
I am afraid the publisher may not understand here.

Reply:
Suggesting instead of the comment put the log below
```suggestion
MoPubLog.log(CUSTOM, ADAPTER_NAME, "onAdViewed - Placement ID " + id);